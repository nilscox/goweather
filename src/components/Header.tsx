/** @jsx jsx */
import { jsx } from '@emotion/core'

type HeaderProps = {
  title: string;
};

const Header = ({ title }: HeaderProps) => (
  <div>
    <h1>{ title }</h1>
  </div>
);

export default Header;
