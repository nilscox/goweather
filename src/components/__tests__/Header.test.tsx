import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';

import Header from '../Header';

test('header', () => {
  const tree = renderer
    .create(<Router><Header /></Router>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
