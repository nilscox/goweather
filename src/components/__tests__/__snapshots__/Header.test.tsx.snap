// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header 1`] = `
<div>
  <nav
    className="navbar navbar-expand-md navbar-light bg-light"
  >
    <a
      aria-current="page"
      className="navbar-brand active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      GoWeather
    </a>
    <button
      className="navbar-toggler"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      style={Object {}}
    >
      <ul
        className="ml-auto navbar-nav"
      >
        <li
          className="nav-item"
        >
          <a
            aria-current={null}
            className="nav-link"
            href="/history"
            onClick={[Function]}
            style={undefined}
          >
            History
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            aria-current={null}
            className="nav-link disabled"
            href="/"
            onClick={[Function]}
            style={undefined}
          >
            About
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            aria-current={null}
            className="nav-link disabled"
            href="/"
            onClick={[Function]}
            style={undefined}
          >
            Log in
          </a>
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
